lib/bipm/data/outcomes.rb

```diff
@@ -0,0 +1,52 @@
+require_relative "outcomes/body"
+require_relative "outcomes/localized_body"
+require_relative "outcomes/meeting"
+
+module Bipm
+  module Data
+    module Outcomes
+      def self.file_path
+        @file_path ||= "#{__dir__}/../../../data/"
```

Comment:
The `{GEM_ROOT}` path may not be writable, for example, in the case in Tebako (read-only filesystem). We should use the home path as default.